eng/bomgenerator/src/main/java/com/azure/tools/bomgenerator/Utils.java

```diff
@@ -183,6 +189,46 @@ static BomDependencyNoVersion toBomDependencyNoVersion(BomDependency bomDependen
     }
 
     static List<BomDependency> parsePomFileContent(Reader responseStream) {
+        List<BomDependency> bomDependencies = new ArrayList<>();
+
+        ObjectMapper mapper = new XmlMapper();
```

Comment:
Any reason to add in Jackson instead of using the previous XML parsing functionality?